doc/rst/source/GMT_Docs.rst

```diff
@@ -707,7 +706,7 @@ modify the appearance of plots or affect the manipulation of data. When
 a program is run, it initializes all parameters to the
 GMT\ defaults [9]_, then tries to open the file ``gmt.conf`` in the current
 directory [10]_. If not found, it will look for that file in a
-sub-directory ``/.gmt`` of your home directory, and finally in your home directory
+sub-directory ``~/.gmt`` of your home directory, and finally in your home directory
```

Comment:
The ~ is unix specific. I think this one should stay as is.

Reply:
Can we change `/.gmt` to `.gmt`?

Reply:
Hmm, why? Doesn't a slash give a more clear indication that .gmt is a directory?

Reply:
OK. I've changed it back to `/.gmt`.